crates/node/builder/src/launch/engine.rs

```diff
@@ -138,11 +138,15 @@ where
         .await?;
 
         // create pipeline
-        let network_client = ctx.components().network().fetch_client().await?;
+        let network_handle = ctx.components().network().clone();
+        let network_client = network_handle.fetch_client().await?;
         let (consensus_engine_tx, consensus_engine_rx) = unbounded_channel();
 
         let node_config = ctx.node_config();
 
+        // We always assume that node is syncing after a restart
+        network_handle.update_sync_state(SyncState::Syncing);
```

Comment:
this is actually also how geth behaves on start, so we should def do this, I can't remember why we did it like this earlier.
I guess because syncing is a bit ambiguous, because on launch it isn't technically syncing just get, because actual sync isnt triggered yet but the appropriate state would be syncing